index.d.ts

```diff
@@ -369,7 +366,8 @@ declare module '@primer/components' {
 
   export interface StateLabelProps extends CommonProps {
     small?: boolean
-    status: 'issueOpened' | 'issueClosed' | 'pullOpened' | 'pullClosed' | 'pullMerged'
+    variant?: 'small' | 'normal'
+    status: 'issueOpened' | 'issueClosed' | 'pullOpened' | 'pullClosed' | 'pullMerged' | 'draft'
```

Comment:
For whatever reason, testing these in the CodeSandbox keeps getting me a TS error for both `variant` and `status` 🤔 

https://codesandbox.io/s/primer-components-ts-testing-47dbw?file=/src/index.tsx:783-844